app/react/portainer/environments/wizard/components/Option/Option.tsx

```diff
@@ -44,11 +45,17 @@ export function Option({
       disabled={isLimited}
       onClick={onClick}
     >
-      <div className="text-center mt-2">
-        {Icon ? (
-          <Icon selected={active} className={styles.iconComponent} />
+      <div className="flex items-end justify-center text-center mt-2">
+        {IconComponent ? (
+          <IconComponent selected={active} className={styles.iconComponent} />
         ) : (
-          <i className={clsx(icon, 'block', styles.icon)} />
+          <Icon
+            icon={icon}
+            className={clsx(
+              'block',
+              icon === 'svg-dockericon' ? styles.iconLarge : styles.icon
```

Comment:
this is too specific. I think in endpoint-types, just import the icon directly, then this L52-58 won't be needed at all.